README.md

```diff
@@ -1,2 +1,136 @@
-# roboconf-dockerfile-dm
-A Dockerfile for Roboconf's DM
+Supported tags and respective `Dockerfile` links
+================================================
+
+- [`5.0`, `5` (*5.0/Dockerfile*)](https://github.com/roboconf/blob/<BLOB_ID>/Dockerfile)
+
+What is Roboconf?
+=================
+
+Roboconf is a distributed solution to deploy distributed applications.  
```

Comment:
I am not sure it is worth adding what is already on the web site.
We will certainly make it sorter.